lib/manageiq/cross_repo/runner.rb

```diff
@@ -57,7 +57,17 @@ def run_tests
     end
 
     def env_vars
-      {"MANAGEIQ_REPO" => core_repo.path.to_s, "BUNDLE_PATH" => bundle_path.to_s, "TEST_SUITE" => test_suite}
+      {
+        "MANAGEIQ_REPO" => core_repo.path.to_s,
+        "BUNDLE_PATH" => bundle_path.to_s,
+        "TEST_SUITE" => test_suite,
+          "CI"                      => "true",
```

Comment:
alignment here is weird - was that intentional?

Reply:
Oh nvm - didn't realize this was WIP.

Reply:
yup just trying to get it to work, haven't figured out a good way of determining the base_ref yet without hitting the github api so for now hard-coded to oparin